routes/users.js

```diff
@@ -0,0 +1,13 @@
+var express = require("express");
```

Comment:
Is this required?

Reply:
Please remove file if not required.

Reply:
ohh sorry its just for practice i will do it in no second.